src/view/com/notifications/FeedItem.tsx

```diff
@@ -467,7 +467,12 @@ let FeedItem = ({
         {item.type === 'feedgen-like' && item.subjectUri ? (
           <FeedSourceCard
             feedUri={item.subjectUri}
-            style={[pal.view, pal.border, styles.feedcard]}
+            style={[
+              t.atoms.bg,
+              t.atoms.border_contrast_low,
+              a.border,
+              styles.feedcard,
+            ]}
```

Comment:
Hi! I'm learning the codebase and am wondering why you switched style objects from `pal. ...` to `t.atoms. ...`?

Reply:
New design system. We're like halfway through migration to the ALF system

Reply:
Thanks!